codemod_yaml/items.py

```diff
@@ -36,6 +36,56 @@ def __eq__(self, other: object) -> bool:
         return other is None or isinstance(other, Null)
 
 
+class Boolean(Item):
```

Comment:
No multiple inheritance from `bool`?

Reply:
Nope, I couldn't figure out how to inherit from bool. Notably tomlkit doesn't either.